platforms/android/build_static_aar.py

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env python
```

Comment:
At first glance this file seem to repeat `build_java_shared_aar.py` in many aspects, is it possible to join these files into a single script with `--static/--dynamic` flag?